### ISSUE TITLE
Generated XLSX generated from Windows system has content issues

### ISSUE THREAD

**Issue by Hackman777** [openspout/openspout#97]
I'm evaluating Openspout to replace PHPExcel on a PHP 8.0.* system. I am creating a very simple test spreadsheet that has text in one row/one cell. The file generates and downloads correctly on the Ubuntu server but the same file generated from my Windows 11 development system has issues with some of the content when loaded. I can tell it to repair and it then appears to load correctly but this is obviously not ideal.

The code I use to generate the file is:

```php
use OpenSpout\Writer\XLSX\Writer;
use OpenSpout\Common\Entity\Cell;
use OpenSpout\Common\Entity\Row;

$writer = new Writer();
$writer->openToBrowser('test.xlsx'); // stream data directly to the browser
$titleRow = new Row([
  Cell::fromValue('First Cell')
]);
$writer->addRow($titleRow);
$writer->close();
```

I have attached 2 files (labeled appropriately): a "good" version generated on the server and a "bad" version generated locally under Windows 11.

FWIW, I had to override the temp folder using sys_temp_dir in php.ini to even get to the point where it would work under Windows. Kept having issues with the c:\windows\temp folder.

Help?

[test_bad.xlsx](https://github.com/openspout/openspout/files/9686220/test_bad.xlsx)
[test_good.xlsx](https://github.com/openspout/openspout/files/9686221/test_good.xlsx)

**Comment by Slamdunk** [comment]
I'm sorry I don't have Windows so I'm not able to give help on this kind of issues

**Comment by spackmat** [comment]
I had a look into the files and it seems to be the problem, that was solved in #87 and Release 4.7.0. Also reported in #85 and #90. What version are you on @Hackman777 ? The issue vanished for me after upgrading to the current 4.7.0.

**Comment by Hackman777** [comment]
I was on release 4.6. I updated to 4.7 and it appears to be working fine now. Thanks, spackmat!
